client/my-sites/email/inbox/mailbox-selection-list/index.jsx

```diff
@@ -51,6 +55,31 @@ const MailboxItemIcon = ( { mailbox } ) => {
 	return null;
 };
 
+const getProvider = ( { mailbox } ) => {
+	if ( isTitanMailAccount( mailbox ) ) {
+		return 'titan';
+	}
+
+	if ( isGoogleEmailAccount( mailbox ) ) {
+		return 'google';
+	}
+
+	if ( isEmailForwardAccount( mailbox ) ) {
+		return 'forward';
+	}
+
+	return null;
+};
```

Comment:
This block of code with hard-coded strings makes me think we really need some constants for these provider strings.

Reply:
I have created constants for Titan and Google, but not for 'forward' as it's not really a provider.